docs/developers/quickstart/proof-composition.md

```diff
@@ -2,9 +2,177 @@
 
 Hylé's native proof verification allows for proof composition. To understand the concept better, we recommend you [read this blog post](https://blog.hyle.eu/proof-composability-on-hyle/).
 
-This guide walks you through creating and deploying your first ticket transfer contract, based on a ticket-app and a simple-token app, by using Hylé and RISC Zero. 
+This guide walks you through creating and deploying your first ticket transfer contract, based on a ticket-app and a [simple-token app](./your-first-smart-contract.md), by using Hylé and RISC Zero.
```

Comment:
I suggest:
[...] creating a contract that used composition with a token transfer contract. We take a basic "ticket-app" as example, and the "simple-token" example